app/main/helpers/validation.py

```diff
@@ -0,0 +1,185 @@
+from functools import reduce
+from operator import add
+import re
+import six
+from werkzeug.datastructures import ImmutableOrderedMultiDict
+
+EMAIL_REGEX = r'^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$'
+
+VALIDATORS = {}
```

Comment:
@tommorris writing a special case class for each framework is a temporary solution, we don't want to be doing this for future frameworks, so hopefully we won't end up with 5 or more classes. At any rate, I'd rather have a single place which listed all available validators than have it spread out throughout the file.